subsys/bluetooth/rpc/host/bt_rpc_internal_host.c

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+
+#include <bluetooth/conn.h>
+#include <hci_core.h>
+
+#include <nrf_rpc_cbor.h>
+
+#include "bt_rpc_common.h"
+#include "serialize.h"
+#include "cbkproxy.h"
+
+#include <logging/log.h>
+
+LOG_MODULE_DECLARE(BT_RPC, CONFIG_BT_RPC_LOG_LEVEL);
+
+static void report_decoding_error(uint8_t cmd_evt_id, void *data)
+{
+	nrf_rpc_err(-EBADMSG, NRF_RPC_ERR_SRC_RECV, &bt_rpc_grp, cmd_evt_id,
+		    NRF_RPC_PACKET_TYPE_CMD);
+}
+
+static void bt_addr_le_is_bonded_rpc_handler(CborValue *value, void *handler_data)
+{
+	printk("host bt_addr_le_is_bonded\n");
```

Comment:
Move this after variable declaration. You should use logger here instead of the `printk`